### ISSUE TITLE
"<string>" is the convention for file-likes with no real filename

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#9016]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#403|
|Type|enhancement|
|Created|2003-11-19 06:19:43Z|

Attachments:

* [ftp_getFPName-patch.txt](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/898/8980dc86c1e9ea324ec99f8b15a2e9a92d0a4d1e/711d3c5edfbb1e053e701c294197a96cd0c42a6c.txt) (745 bytes) - added by exarkun on 2003-11-19 06:20:04Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__403 403
type__enhancement enhancement
reporter__exarkun exarkun
priority__low low
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__ 
keywords__ 
time__1069222783000000 1069222783000000
changetime__1469749172783638 1469749172783638
version__ 
owner__slyphon slyphon
cc__exarkun cc__slyphon
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun commented|
|-|-|

```
#!html
<pre>
Patch attached
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="slyphon's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|slyphon commented|
|-|-|

```
#!html
<pre>
good show ol' chap.

i like working with exarkun. apply the patch, it works better than my code did,
and then you commit. :)
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|GitHub commented|
|-|-|

In changeset 4e225c70e9f24af4166fb625d2c0d6645f159c71
```
#!CommitTicketReference repository="" revision="4e225c70e9f24af4166fb625d2c0d6645f159c71"
Merge pull request #403 from twisted/8666-rodrigc-text

Author: glyph
Reviewer: adiroiban, rodrigc
Fixes: #8666

Fix test_text.py on Python 3
```
